docs/internal/templates/specifications/tests/TestCaseIdentifierSpecification.md

```diff
@@ -13,25 +13,26 @@ test level's name. The following test levels are defined and available:
 * Syntax
 * Zone
 
-The test level name is not case sensitive.
+The module name is not case sensitive, but camel case, as defined above,
+should be prefered.
 
 The test case identifier in the test case specification (both in the main
-headline and in the "Test case identifier" section) uses the test level name
-in all uppercase and has the format: `{Test Level name} + {Index}`
+headline and in the "Test case identifier" section) uses the module name,
+as defined above, and has the format: `{Module name} + {Index}`
 
-When referencing to a test case the test level name may be used in other letter
-case than all uppercase.
+When referencing to a test case the module name may be used in other letter
+case, but for readability, camel case sould be prefered.
```

Comment:
I do not think these are examples of "camelCase", https://en.wikipedia.org/wiki/Camel_case

"sould" --> "should"

Reply:
No, "Address" is not PascalCase.

Reply:
See my other comment.